DESCRIPTION

```diff
@@ -10,3 +10,4 @@ Authors@R:
 Description: Collection of technical guidelines.
 Imports: 
     bookdown
+License: LGPL (>= 2.1)
```

Comment:
@Chargothrond, I guess you used `LGPL (>= 2.1)` as we have in ShinyCICD (where it might have originally come from `golem`). We would typically use `GPL-3`... Please confirm with @RolandASc, then delete this field + license file(s), and finally restart with `usethis::use_gpl3_license()` (or any agreed license).